notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2216-plots.ipynb

```diff
@@ -0,0 +1,1166 @@
+    "For the hexapods, the compensation mode is disabled. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92121b54-0e97-41fe-92b0-bbef54d51f09",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# t_start = \"2022-07-20 19:18:09.246021\"\"\n",
+    "# t_end = \"2022-07-20 19:22:07.383914\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd29520d-e70b-4adb-b54b-dca7a7059947",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = await vandv.efd.query_script_message_contains(\n",
+    "    client=client, \n",
+    "    contains=[test_case, test_exec, \"START\", \"Slewing without LUT\"],\n",
+    "    upper_t=Time.now(),\n",
+    "    lower_t=Time(\"2022-06-01 17:33\", scale=\"utc\", format=\"iso\"),\n",
+    "    num=1)\n",
+    "\n",
+    "t_start = vandv.efd.time_pd_to_astropy(df.index[0].floor(\"1s\"))\n",
+    "print(t_start, df.iloc[0][\"message\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c46fc5b-f750-43a4-8b0c-633f04173853",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "df = await vandv.efd.query_script_message_contains(\n",
+    "    client=client, \n",
+    "    contains=[test_case, test_exec, \"END\", \"Slewing without LUT\"],\n",
+    "    upper_t=Time.now(),\n",
+    "    lower_t=Time(\"2022-06-01 17:33\", scale=\"utc\", format=\"iso\"),\n",
+    "    num=1)\n",
+    "\n",
+    "t_end = vandv.efd.time_pd_to_astropy(df.index[0].ceil(\"1s\"))\n",
+    "print(t_end, df.iloc[0][\"message\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c21f65ea-da47-4612-9846-aa9da06e7479",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(f\"Starting data analysis between\\n  {t_start}\\n    and\\n  {t_end}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cd8a712-02ce-4427-aad8-0a8c428e8100",
+   "metadata": {},
+   "source": [
+    "### Analyse M1M3 Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eaaa7d8c-5c07-45f3-be20-5d0eb4482a81",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fad = await client.select_time_series(\n",
+    "    \"lsst.sal.MTM1M3.forceActuatorData\", \n",
+    "    \"*\", \n",
+    "    t_start,\n",
+    "    t_end)\n",
+    "\n",
+    "el = await client.select_time_series(\n",
+    "    \"lsst.sal.MTMount.elevation\",\n",
+    "    \"*\", \n",
+    "    t_start,\n",
+    "    t_end)\n",
+    "\n",
+    "fig, axs = plt.subplot_mosaic(\n",
+    "    mosaic=\"A\",\n",
+    "    num=\"Slew Without Correction\", \n",
+    "    constrained_layout=True,\n",
+    "    dpi=120,\n",
+    "    figsize=(12, 4),\n",
+    ")\n",
+    "\n",
+    "fig.suptitle(\"M1M3 Timeline - Slew LUT+inclinometer\")\n",
```

Comment:
Consider either changing the title here `Slew LUT+inclinometer`, or the title of the section `Slewing without LUT`. (Is it without LUT or with LUT)?

Reply:
Done.